src/main.rs

```diff
@@ -84,13 +91,20 @@ async fn main() -> Result<(), Box<dyn Error>> {
         .init();
 
     let cli = Cli::parse();
+    let mut middleware: Option<Arc<Vec<MiddlewareFn>>> = None;
+
     match cli.command {
         Commands::Run {
             headers,
             mut args,
             env_vars,
             publish,
+            security,
         } => {
+            if security {
+                const BASE_URL: &str = "http://127.0.0.1:8081";
```

Comment:
Is this fixed all the time, or can it be configured?

Reply:
It's easy to add this on the CLI side, but I don't have a clear idea about the parameter format yet. I'd like to review this fixed version first and make adjustments in a separate commit later.